cfgov/unprocessed/apps/teachers-digital-platform/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "teachers-digital-platform",
+  "description": "Youth financial education activity search and building blocks journey tool",
+  "homepage": "http://localhost:8000/practitioner-resources/youth-financial-education/",
```

Shouldn't the homepage point to cfgov, as other apps do?

```suggestion
  "homepage": "https://www.consumerfinance.gov/practitioner-resources/youth-financial-education/",
```

---

Ah, good catch!